tools/Customization/DevHome.Customization/Views/DeveloperFileExplorerPage.xaml.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.ObjectModel;
+using DevHome.Common.Extensions;
+using DevHome.Common.Services;
+using DevHome.Customization.ViewModels;
+using DevHome.Settings.Models;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+
+namespace DevHome.Customization.Views;
+
+public sealed partial class DeveloperFileExplorerPage : Page
+{
+    public DeveloperFileExplorerViewModel ViewModel
+    {
+        get;
+    }
+
+    public ObservableCollection<Breadcrumb> Breadcrumbs
+    {
+        get;
+    }
+
+    public DeveloperFileExplorerPage()
+    {
+        ViewModel = Application.Current.GetService<DeveloperFileExplorerViewModel>();
+        this.InitializeComponent();
+
+        var stringResource = new StringResource("DevHome.Customization/Resources");
+        Breadcrumbs = new ObservableCollection<Breadcrumb>
```

Comment:
Just as an FYI, if you wanted this stuff in the ViewModel, you can do the same thing we did in #2313 to move it over and use the built-in headers. But you might want to hold off on that, since we're considering changing the app-wide header template to be a BreadcrumbBar (input welcome).